be/src/cloud/pb_convert.cpp

```diff
@@ -287,6 +287,7 @@ void doris_tablet_schema_to_cloud(TabletSchemaCloudPB* out, const TabletSchemaPB
     out->set_inverted_index_storage_format(in.inverted_index_storage_format());
     out->set_enable_variant_flatten_nested(in.variant_enable_flatten_nested());
     out->set_skip_bitmap_col_idx(in.skip_bitmap_col_idx());
+    out->set_storage_page_size(in.storage_page_size());
```

Comment:
I suggest this property storing in TabletMetaPB

Reply:
done